internal/handlers/msg_comparison_test.go

```diff
@@ -0,0 +1,345 @@
+package handlers
+
+import (
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/testutil"
+	"github.com/stretchr/testify/assert"
```

Comment:
Imports should be grouped; check other files

Reply:
Fixed.